apps/web/app/_utils.tsx

```diff
@@ -74,9 +66,31 @@ export const _generateMetadata = async (
       url: canonical,
       type: "website",
       siteName: APP_NAME,
-      title,
-      images: [image],
+      title: displayedTitle,
     },
     metadataBase,
   };
 };
+
+export const _generateMetadata = async (
+  getTitle: (t: TFunction<string, undefined>) => string,
+  getDescription: (t: TFunction<string, undefined>) => string,
+  hideBranding?: boolean,
+  origin?: string
+) => {
+  const metadata = await _generateMetadataWithoutImage(getTitle, getDescription, hideBranding, origin);
+  const image =
+    SEO_IMG_OGIMG +
+    constructGenericImage({
```

Comment:
In upcoming PRs, I will create `generateMeetingMetadata` and `generateAppMetadata`. They will use `constructMeetingImage` and `constructAppImage` respectively.